src/friend_area_action_menu.c

```diff
@@ -449,11 +440,11 @@ void sub_80278B4(void)
         SetFriendAreaActionMenuState(FRIEND_AREA_ACTION_MENU_MAIN_2);
         break;
       case FRIEND_AREA_ACTION_MENU_ACTION_MAKE_LEADER:
-        pokeStruct2 = &gRecruitedPokemonRef->pokemon[sUnknown_203B2BC->targetPoke];
+        newLeader = &gRecruitedPokemonRef->pokemon[sUnknown_203B2BC->targetPoke];
         playerStruct = GetPlayerPokemonStruct();
-        if (!pokeStruct2->isTeamLeader) {
+        if (!newLeader->isTeamLeader) {
           playerStruct->isTeamLeader = FALSE;
-          pokeStruct2->isTeamLeader = TRUE;
+          newLeader->isTeamLeader = TRUE;
           nullsub_104();
```

Comment:
It'd be cool to make a comment at nullsub_104 stating it's called when the team leader is changed by the player. Eventually when we are at documentation, it'll be cool to see what all the nullsubs were used for (probably printing or breakpoints)

Reply:
Would be cool to see if the nullsubs are actually used in JP or Blue versions.